packages/form-builder/src/feedback/FeedbackIcon.tsx

```diff
@@ -1,7 +1,19 @@
-const FeedbackIcon = (props) => {
+const FeedbackIcon = () => {
     return (
-        <svg {...props} width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <path d="M11.3327 11.4993H2.66602L0.666016 13.4993V2.16602C0.666016 1.06602 1.56602 0.166016 2.66602 0.166016H11.3327C12.4327 0.166016 13.3327 1.06602 13.3327 2.16602V9.49935C13.3327 10.5993 12.4327 11.4993 11.3327 11.4993Z" fill="white"/>
+        <svg
```

Comment:
I tried to import this as an svg file and assign to an img src but it kept making a relative url as the src instead of an inline data src.  This works fine in the next gen app but not here.  Must be a webpack config this since we're using laravel mix here.  Should figure that out at some point for the future when using images / local icons.